src/components/FooterCanvas/FooterCanvas.component.tsx

```diff
@@ -5,6 +5,10 @@ interface IStyledCanvas {
   ref?: any;
 }
 
+type ExtendedHTMLCanvasElement = HTMLCanvasElement & {
+  getContext: any,
```

Comment:
If you hover the getContext I think you'll be able to see the type but let's talk about it later don't worry